test/runtests.jl

```diff
@@ -25,8 +25,8 @@ const mgsr = ModifiedGramSchmidtIR(η₀)
 # Tests
 # -----
 t = time()
-include("factorize.jl")
-include("gklfactorize.jl")
+# include("factorize.jl")
+# include("gklfactorize.jl")
```

Comment:
This is presumably accidental?
```suggestion
include("factorize.jl")
include("gklfactorize.jl")
```

Reply:
Yes. That's me skipping over to the relevant tests 😄